src/components/chatbox/chatbox.tsx

```diff
@@ -0,0 +1,550 @@
+import React, { useEffect, useState, useRef } from 'react'
```

Comment:
### What?

Renders the following:

![Uploading CleanShot 2023-09-08 at 11.29.29@2x.png…]()